plugins/module_utils/network/nxos/facts/static_routes/static_routes.py

```diff
@@ -35,6 +35,10 @@ def __init__(self, module):
 
     def get_static_routes_data(self, connection):
         non_vrf_data = connection.get("show running-config | include '^ip(v6)* route'")
+        if non_vrf_data:
+            non_vrf_data = "\n".join(
+                line for line in non_vrf_data.splitlines() if " vrf " not in line
+            )
         vrf_data = connection.get("show running-config | section '^vrf context'")
         if vrf_data:
             non_vrf_data += "\n" + vrf_data
```

Comment:
when there are no global routes, non_vrf_data stays None and the += will leads to crash